### PR TITLE
Update .asf.yaml

### PR DESCRIPTION
**Pull request by ajantha-bhat**

### CODE REVIEW

File: .asf.yaml
Comment (ajantha-bhat) [inline .asf.yaml:52]: @findepi: Congratulations on being committer :) 

I would also like to remove the people who are inactive from 6 months or more in iceberg repo from this list to make space for new people who are active and intersted in project management. What do you guys think?

Comment (ajantha-bhat) [reply .asf.yaml:52]: And @jbonofre: Since you are more active on the release side, I think we should add you here :)

Comment (jbonofre) [reply .asf.yaml:52]: @ajantha-bhat 😄 Do you want to add me in this PR or do you want me to create another PR ?

Comment (jbonofre) [reply .asf.yaml:52]: I agree with the 6 months period. It sounds reasonable to me.

Comment (Fokko) [reply .asf.yaml:52]: Good one @ajantha-bhat, and I'm all in for adding @jbonofre to the list 👍

**Review by Fokko — APPROVED** [review]

**Review by findepi — APPROVED** [review]

**Comment by findepi** [comment]
@ajantha-bhat this PR probably reduces my access to the repo currently. Would you be willing to merge it once I actually have the committer access. We can test it eg by me merging (which it seems I cannot yet)

**Comment by ajantha-bhat** [comment]
> @ajantha-bhat this PR probably reduces my access to the repo currently. Would you be willing to merge it once I actually have the committer access. We can test it eg by me merging (which it seems I cannot yet)

@findepi: Sure. I assumed we give privileges before announcing on the mailing list. Feel free to merge this PR when you get the access. No hurry.

**Review by hussein-awala — APPROVED** [review]

**Comment by findepi** [comment]
@ajantha-bhat thanks for your patience. Looks we're all set. 
And this is definitely the best first PR to merge :)

**Pull request merged**
